src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Mappings.cs

```diff
@@ -16,12 +16,19 @@ public static class Mappings
                 AgreedOn = source.AgreedOn,
             };
 
-        public static ApprenticeshipUpdated ToApprenticeship(this ApprenticeshipUpdatedApproved2Event source) =>
+        public static ApprenticeshipUpdated ToApprenticeshipUpdated(this ApprenticeshipCreated2Event source) =>
             new ApprenticeshipUpdated
             {
-                ApprenticeshipId = source.ApprenticeshipId,
-                ApprovedOn = source.ApprovedOn,
-                Email = source.Email,
+                ContinuationOfCommitmentsApprenticeshipId = source.ContinuationOfId,
+                CommitmentsApprenticeshipId = source.ApprenticeshipId,
+                CommitmentsApprovedOn = source.AgreedOn,
+            };
+
+        public static ApprenticeshipUpdated ToApprenticeshipUpdated(this ApprenticeshipUpdatedApprovedEvent source) =>
+            new ApprenticeshipUpdated
+            {
+                CommitmentsApprenticeshipId = source.ApprenticeshipId,
+                CommitmentsApprovedOn = source.ApprovedOn,
```

Comment:
I think that if we're going to change these names (which think is a decent idea) then we should do it for the existing `Created` DTO as well, otherwise it's confusing why the update has the `Commitments` prefix and the create doesn't.

Reply:
agreed, but didn't want to extend the scope of this story, as I knew you were refactoring that

Reply:
Did this change and fixed the date to use CreatedOn and not AgreedOn